File: app/src/main/java/ru/application/homemedkit/connectionController/models/Foiv.java
package ru.application.homemedkit.connectionController.models;

import static ru.application.homemedkit.helpers.ConstantsHelper.BLANK;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonSetter;
import com.fasterxml.jackson.annotation.Nulls;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Foiv {
    @JsonProperty("prodFormNormName")
    @JsonSetter(nulls = Nulls.SKIP)
    public String prodFormNormName = BLANK;
    @JsonProperty("prodDNormName")
    @JsonSetter(nulls = Nulls.SKIP)
    public String prodDNormName = BLANK;
}

File: app/src/main/java/ru/application/homemedkit/connectionController/models/Foiv.kt
package ru.application.homemedkit.connectionController.models

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import ru.application.homemedkit.helpers.BLANK

@JsonClass(generateAdapter = true)
data class Foiv(
    @Json val prodFormNormName: String = BLANK,
    @Json val prodDNormName: String? = BLANK,
    @Json val prodPack1Size: String? = BLANK
)
